doc/man/8/ceph.rst

```diff
@@ -1111,6 +1111,53 @@ Usage::
 
   ceph osd ok-to-stop <id> [<ids>...] [--max <num>]
 
+Subcommand The subcommand ``ok-to-upgrade`` determines a safe set of OSD(s)
```

Comment:
`Subcommand The subcommand [...]` should probably be just `The subcommand [...]`.

Reply:
Okay, I have changed it to how it is for other subcommands in the document to maintain consistency shown below:

`Subcommand ok-to-upgrade [...]`